nemo/collections/asr/models/classification_models.py

```diff
@@ -349,75 +364,66 @@ def transcribe(self, paths2audio_files: List[str], batch_size: int = 4, logprobs
 
             A list of transcriptions (or raw log probabilities if logprobs is True) in the same order as paths2audio_files
         """
-        if paths2audio_files is None or len(paths2audio_files) == 0:
-            return []
-        # We will store transcriptions here
+        if override_config is None:
+            trcfg = ClassificationTranscribeConfig(batch_size=batch_size, logprobs=logprobs)
+        else:
+            if not isinstance(override_config, ClassificationTranscribeConfig):
+                raise ValueError(
+                    f"override_config must be of type {ClassificationTranscribeConfig}, "
+                    f"but got {type(override_config)}"
+                )
+            trcfg = override_config
+
+        return super().transcribe(audio=audio, override_config=trcfg)
+
+    """ Transcription related methods """
+
+    def _transcribe_input_manifest_processing(
+        self, audio_files: List[str], temp_dir: str, trcfg: ClassificationTranscribeConfig
+    ):
+        with open(os.path.join(temp_dir, 'manifest.json'), 'w', encoding='utf-8') as fp:
+            for audio_file in audio_files:
+                label = 0.0 if self.is_regression_task else self.cfg.labels[0]
```

Comment:
instead of first label, why not add `infer` as label class.

Reply:
Cause it's read from config inside data loader - if infer not in config it crashes

Reply:
yeah, it fails for older models from speaker models but should be fine for all classification models?

Reply:
I dont wanna take a chance.